### PR TITLE
Fix last modified color and add playlist button

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Fix last modified color and add playlist button

[stage-19]

## Motivation and Context
Fix for #1924 

## How Has This Been Tested?
Tested updates visually in the local environment. No functionality changes.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: web/partials/schedules/schedule-fields.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
           <label translate>schedules-app.playlist.empty</label>
         </div>
         <div class="btn-group mb-2" dropdown>
-          <button id="addPlaylistItemButton" type="button" dropdown-toggle class="dropdown-toggle btn btn-default btn-toolbar">{{'schedules-app.playlist.add-playlist-item' | translate}}</button>
+          <button id="addPlaylistItemButton" type="button" dropdown-toggle class="btn btn-default btn-toolbar">{{'schedules-app.playlist.add-playlist-item' | translate}}</button>
```
Comment (ezequielc) [inline web/partials/schedules/schedule-fields.html:61]: @alex-deaconu I think the dropdown-toggle directive is adding back the removed class. I still see the shadow when I click the button. Thanks!

Comment (alex-deaconu) [reply web/partials/schedules/schedule-fields.html:61]: @ezequielc good catch, will fix. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc fixed by explicitly setting `box-shadow: none`. Please review the last commit. Thanks!

**Pull request merged**